src/js/visual/configurator-prc.js

```diff
@@ -12,6 +12,19 @@ if (
     var ripe = base.ripe;
 }
 
+/**
+ * Binds an PRC Configurator to this Ripe instance.
+ *
+ * @param {Configurator} element The PRC Configurator to be used by the Ripe instance.
+ * @param {Object} options An Object with options to configure the Configurator instance.
+ * @returns {Configurator} The Configurator instance created.
+ */
+ripe.Ripe.prototype.bindConfiguratorPrc = function(element, options = {}) {
+    options = Object.assign({}, { format: this.format }, options);

```

Comment:
You are doing this twice, both on `ripe.Ripe.prototype.bindConfigurator` and here.
You should probably remove the one from `ripe.Ripe.prototype.bindConfigurator`

Reply:
I'd rather keep that one there, and remove the one from `bindConfiguratorPrc`, since both CSR and PRC could use the same options that are being passed here. Another nice catch, fixing!